cmd/crictl/sandbox.go

```diff
@@ -0,0 +1,225 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/urfave/cli"
+	"golang.org/x/net/context"
+	pb "k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
+)
+
+var runPodSandboxSpec string
+var podSandboxID string
+var filter string
+
+var runtimeRunPodSanboxCommand = cli.Command{
+	Name:  "runsandbox",
+	Usage: "run a sandbox",
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:        "podsandbox-spec",
```

Comment:
The flag seems too long. Could we make it just an arg without flag, e.g. `crictl sandbox run config.json|yaml`?